src/container/HomeContainer/index.jsx

```diff
@@ -3,24 +3,78 @@ import styled from 'styled-components';
 
 import dayBg from 'assets/images/day_bg.jpg';
 import nightBg from 'assets/images/night_bg.jpg';
+import buildingLight_1 from 'assets/images/building_light1.png';
+import buildingLight_2 from 'assets/images/building_light2.png';
 
 import {MainContent, HeaderContainer} from 'container';
 
 
 const HomeLayout = styled.div`
   width: 100%;
   height: 100vh;
-  background: url(${props => props.backgroundBg});
+  background: url(${props => props.backgroundBg ? dayBg : nightBg});
   background-size: cover;
+  position: relative;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background: url(${buildingLight_1});
+    background-size: cover;
+    opacity: ${props => props.backgroundBg? 0 : 1};
+    pointer-events: none;
+  }
+  &::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background: url(${buildingLight_2});
+    background-size: cover;
+    opacity: ${props => props.backgroundBg? 0 : 1};
+    pointer-events: none;
+  }
+
 `
 
 const HomeContainer = () => {
   const [ dayTime, setDayTime] = useState(true);
+  const [ soliloquy, setSoliloquy] = useState({txt: '가자가자', count: 0});
+  const talk = {
+    day: ['출근길이 막히네', '이제 출근하지만 벌써 집가고싶다...', '졸려', '. . .', '좋은 아침~'],
+    night: ['칼퇴근이다', '집까지 5분', '야식 뭐먹지', '길 안막혀서 좋다', '음주운전, 졸음운전 금지'],
+  }
+
+  const changeDayAndNight = () => {
+    setDayTime(dayTime? false : true)
```

Comment:
이 부분은 좀 더 simplied 될 수 있을 것 같아요!! 근데 개인적으로 이거는 편한것과 가독성을 생각하면서 사용하면 될 것 같아요!
```suggestion
    setDayTime(!dayTime)
```

Reply:
헛... 아! 
저 상태가 true 면 false, false면 true 해야지!
라고만 생각했어서 나온 코드에요

생각해보니 ! <- 이게 반대로 만들어주니까 저렇게 작성하는게 더 깔끔하겠어요